Clustering/Hdbscan/src/main/java/org/tribuo/clustering/hdbscan/HdbscanTrainer.java

```diff
@@ -771,6 +809,7 @@ public TrainerProvenance getProvenance() {
         private final Integer label;
         private final Double outlierScore;
         private final SGDVector features;
+        private Double maxDistToEdge = Double.NEGATIVE_INFINITY;
```

Comment:
Can this be final, then modify the get to return Double.NEGATIVE_INFINITY if the field is null? That will make it easier to expose when we make the exemplars public later.

Reply:
Sure, I've updated the code to construct the `ClusterExemplar`s later on, which enables this variable to be final.